### ISSUE TITLE
Can't use ping with mullvad-exclude

### ISSUE THREAD

**Issue by Juerd** [mullvad/mullvadvpn-app#4677]
## Issue report

Operating system: Linux

App version: 2023.3

### Issue description

With `mullvad-exclude`, ping doesn't work, because the incoming echo replies are dropped by the input chain.

The following will fix it temporarily:

```
nft add rule inet mullvad input icmpv6 type echo-reply accept
nft add rule inet mullvad input icmp type echo-reply accept
```

It would be nice if the app did this automatically, either by default or via a configuration option. I don't see any security or privacy downside of allowing incoming echo replies (but if there are any, I'd love to learn!)

**Comment by pinkisemils** [comment]
Hiya, could you elaborate on the particular Linux you're running - we're struggling to reproduce this. We'd also be OK with just improving the firewall code to help with this. You are correct in assessing that allowing echo responses shouldn't have an impact on privacy or security.

**Comment by Ayymoss** [comment]
Did it seem that your pings would time out after a while? Let's say the DNS cache expired or something.

I've been facing what I want to say is a similar issue, where all DNS queries will fail to resolve after an extended period, approx 1 hour—disconnecting from Mullvad and reconnecting resolves it. 

However, today it has been fine for the last few hours. I'm a bit perplexed.

**Comment by Juerd** [comment]
No, the problem I had with not being able to receive pings via mullvad-exclude was not related to DNS; I was pinging an IP address directly.

I did have many issues with the Mullvad DNS servers: at times, resolving is very slow, sometimes even timing out; I switched to some public nameservers because it was not workable and I didn't feel like trying to debug the issue. I can't see any way in which anything with DNS could have caused the issue with not being able to ping an IP address, though, so I think these are different issues.

**Comment by Ayymoss** [comment]
Alright, thanks for the quick reply. Good luck with your resolve. :)

**Comment by Juerd** [comment]
> Can't reboot right now, but I'll let you know what that does when I get around to it.

Even after rebooting, I have not been able to reproduce the issue.

With no way of reproducing the issue, and thus no way of being certain anything is actually wrong, maybe it's better to close this issue.

**Comment by faern** [comment]
I'm closing, as this should not really be an issue. And no one can currently reproduce it.

**Comment by saber1440** [comment]
Hi,
I just happened to have this issue today. 
OS: Pop!_OS - 22.04
App version: 2023-04 and 2023-05-beta1

The issue is that if I am connected to a server, I can get a response from `mullvad-exclude ping <address>`. But when not connect to a server AND in the Lock-down mode, running `mullvad-exclude ping <address>` does not return anything. 

I was able to do this before and connect using [bridge and ssh](https://mullvad.net/en/help/ssh-and-mullvad-vpn/). But that's not possible anymore.

**Comment by saber1440** [comment]
Hi,
I just happened to have this issue today. 
OS: Pop!_OS - 22.04
App version: 2023-04 and 2023-05-beta1

The issue is that if I am connected to a server, I can get a response from `mullvad-exclude ping <address>`. But when not connect to a server AND in the Lock-down mode, running `mullvad-exclude ping <address>` does not return anything. 

I was able to do this before and connect using [bridge and ssh](https://mullvad.net/en/help/ssh-and-mullvad-vpn/). But that's not possible anymore.
